docs/source/user_guide/input_output.md

```diff
@@ -1,31 +1,45 @@
 (target-io)=
 # Input/Output
 
-(target-formats)=
-## Supported formats
-(target-supported-formats)=
-`movement` supports the analysis of trajectories of keypoints (_pose tracks_) and of bounding box centroids (_bounding box tracks_).
+## Overview
+
+Our goal with `movement` is to enable pipelines that are input-agnostic,
+meaning they are not tied to a specific motion tracking tool or data format.
+Therefore, our input/output functions are designed to facilitate data flows
+between various third-party formats and `movement`'s own native
+[data structure](target-poses-and-bboxes-dataset) based on [xarray](xarray:).
```

Comment:
For some reason I find that the bullet points come out of nowhere, so something like this may help it flow better:
```suggestion
[data structure](target-poses-and-bboxes-dataset) based on [xarray](xarray:). It may be useful to think of movement supporting two types of data loading/saving:
```